efiction/metadata.py

```diff
@@ -27,6 +27,7 @@ def __init__(self, config: ConfigParser, logger: Logger, sql: SqlDb, step_path:
         self.tag_converter = TagConverter(config, logger, sql)
         self.authors = []
         self.ratings = []
+        self.ratings_dirty: bool = False
```

Comment:
Haha, given the nature of some of the contents we handle, and that we wanted to move away from terminology like "sausage machine", maybe there's a better name to use here. Perhaps simply `rating_is_string`? (and change the naming in the TagConverter too)

Reply:
And forgot to say that this could also do with a unit test to make sure it works, and doesn't regress when we next tinker with the code.